src/equations/ideal_glm_mhd_multicomponent_2d.jl

```diff
@@ -711,4 +711,9 @@ end
     return SVector{ncomponents(equations), real(equations)}(u[i + 8] * v
                                                             for i in eachcomponent(equations))
 end
+
+# Return the magnetic field
+function magnetic_field(u, equations::IdealGlmMhdMulticomponentEquations2D)
+    SVector(u[5], u[6], u[7])
+end
```

Comment:
```suggestion
magnetic_field(u, equations::IdealGlmMhdMulticomponentEquations2D) = SVector(u[5], u[6], u[7])
```
For consistency?

Reply:
This is actually how I had it before, but the VS formatter converted it to the current version :rofl: (the line is too long). Should I then keep the long version for both equations?